packages/synapse-react-client/test/lib/containers/oauth/CreateOAuthClient.integration.test.tsx

```diff
@@ -52,8 +52,8 @@ function setUp(props: CreateOAuthModalProps = defaultProps) {
   const user = userEvent.setup()
   const component = renderComponent(props)
   const inputs = {
-    name: screen.getByLabelText('Client Name'),
-    homePage: screen.getByLabelText('Client Homepage'),
+    name: screen.getByRole('textbox', { name: 'Client Name' }),
+    homePage: screen.getByRole('textbox', { name: 'Client Homepage' }),
```

Comment:
Strange that our MUI TextField isn't giving us accessible labels...I wonder if that's a bug on our end.

Reply:
I think when a field is required, there is an asterisk span included in the label, although it's hidden from the accessibility API (`aria-hidden="true"`) -- 

```
<label
  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined css-199xmkx-MuiFormLabel-root-MuiInputLabel-root"
  data-shrink="true"
  for="clientName"
  id="clientName-label"
>
    Client Name
    <span
      aria-hidden="true"
      class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-1ygj4mw-MuiFormLabel-asterisk"
    >                     
      *
    </span>
</label>
```

In cases where the label text is broken up, [react-testing-library](https://testing-library.com/docs/queries/bylabeltext#name) recommends using getByRole with a name --

> The example above does NOT find the input node for label text broken up by elements. You can use getByRole('textbox', { name: 'Username' }) instead which is robust against switching to aria-label or aria-labelledby.

We could still find "Client Homepage" by label text though, since it's not a required field. I'll make that change in the next commit!